File: PY/1985.py
# BEE 1985 - MacPRONALTS
def calcular(cod, q):
    cardapio = dict({"1001": 1.5,"1002": 2.5,"1003": 3.5,"1004": 4.5,"1005": 5.5})

    return q * cardapio[str(cod)]

p = int(input())
r = 0

for _ in range(p):
    cod, q = map(int, input().split())
    r += calcular(cod, q)

print("%.2f" %r)

File: C/1985.c
// BEE 1985 - MacPRONALTS
#include <stdio.h>

float calcular(int cod, int q)
{
    switch (cod)
    {
    case 1001:
        return q * 1.5;
    case 1002:
        return q * 2.5;
    case 1003:
        return q * 3.5;
    case 1004:
        return q * 4.5;
    case 1005:
        return q * 5.5;
    default:
        return 0;
    }
}

int main()
{
    int p, cod, q;
    float r = 0;

    scanf("%d", &p);
    for (int i = 0; i < p; i++)
    {
        scanf("%d %d", &cod, &q);
        r += calcular(cod, q);
    }

    printf("%.2f\n", r);

    return 0;
}